libkbfs/folder_block_manager.go

```diff
@@ -504,6 +490,35 @@ func (fbm *folderBlockManager) archiveBlocksInBackground() {
 	}
 }
 
+func (fbm *folderBlockManager) deleteBlocksInBackground() {
+	for {
+		select {
+		case toDelete := <-fbm.blocksToDeleteChan:
+			fbm.runUnlessShutdown(func(ctx context.Context) (err error) {
+				if err := fbm.processBlocksToDelete(ctx, toDelete); err != nil {
```

Comment:
Just like the archiver, I think you'll want to make a new context with a cancel function here, and cancel it in the `shutdown()` method, to make sure any deletes happening in the background get shut down when a test is over.  Limiting it to the `backgroundTaskTimeout` is probably a good idea too.